src/networks.json

```diff
@@ -2118,6 +2118,22 @@
     "start": 853908,
     "logo": "ipfs://QmVctLQ44vhkwejja9DDjDYUdYgVRBEWs242mhd95SeM5q"
   },
+  "1978": {
+    "key": "1978",
+    "name": "BUTANE Mainnet ",
+    "shortName": "Core",
+    "chainId": 1978,
+    "network": "mainnet",
+    "multicall": "0x024f0041b76B598c2A0a75004F8447FaF67BD004",
```

Comment:
Hey @bbcchain Looks like this is not a multicall? `0x024f0041b76B598c2A0a75004F8447FaF67BD004` 
is there some way to check source code of this contract?